phoenix-core-server/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -1388,6 +1398,18 @@ public void postBatchMutateIndispensably(ObserverContext<RegionCoprocessorEnviro
       try {
           if (success) {
               context.currentPhase = BatchMutatePhase.POST;
+              if(context.hasAtomic && miniBatchOp.size() == 1) {
+                  if (miniBatchOp.getOperationStatus(0).getResult() == null) {
+                      byte[] retVal = PInteger.INSTANCE.toBytes(1);
+                      Cell cell = PhoenixKeyValueUtil.newKeyValue(
+                              miniBatchOp.getOperation(0).getRow(),
+                              EMPTY_BYTE_ARRAY, EMPTY_BYTE_ARRAY, 0, retVal, 0,
```

Comment:
We need to give the column family and column qualifier

Reply:
+1

Reply:
yea I neglected this place, will change

Reply:
Done